ui/constants/notifications.js

```diff
@@ -5,6 +5,7 @@ export const RULE = {
   FIAT_TIP: 'fiat_tip',
   NEW_CONTENT: 'new_content',
   NEW_LIVESTREAM: 'new_livestream',
+  WEEKLY_WATCH_REMINDER: 'weekly_watch_reminder',
```

Comment:
Since unrelated to theme, move this into another PR to narrow down the scope?

Reply:
The current master links to a "subscriptions" page which does not exist. When I see an obvious mistake or oversight I just fix it instantly. A separate PR seems kinda pointless.

Reply:
@toshokanneko a separate PR isn't pointless at all. Each PR should be as focused as possible on one task.

Reply:
At the time of that comment I was under the impression that the merging process for the new themes is imminent. That's why I thought that it would be pointless, but I get your point. I'll make separate PRs in cases like this from now on.